rel-eng/releasers.conf

```diff
@@ -1,45 +1,45 @@
 # Foreman Releasers
 [koji-foreman]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-nightly-nonscl-rhel7 foreman-nightly-rhel7 foreman-nightly-el8
+autobuild_tags = foreman-3.2-nonscl-rhel7 foreman-3.2-rhel7 foreman-3.2-el8
 builder.rpmbuild_options = --define "foremandist .fm3_2"
 
 [koji-foreman-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-nightly-nonscl-rhel7 foreman-nightly-rhel7 foreman-nightly-el8
+autobuild_tags = foreman-3.2-nonscl-rhel7 foreman-3.2-rhel7 foreman-3.2-el8
 builder = tito.builder.FetchBuilder
 builder.jenkins_url = https://ci.theforeman.org
 builder.rpmbuild_options = --define "foremandist .fm3_2"
 
 [koji-foreman-plugins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-plugins-nightly-nonscl-rhel7 foreman-plugins-nightly-rhel7 foreman-plugins-nightly-el8
+autobuild_tags = foreman-plugins-3.2-nonscl-rhel7 foreman-plugins-3.2-rhel7 foreman-plugins-3.2-el8
 builder.rpmbuild_options = --define "foremandist .fm3_2"
 
 [koji-foreman-plugins-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-plugins-nightly-nonscl-rhel7 foreman-plugins-nightly-rhel7 foreman-plugins-nightly-el8
+autobuild_tags = foreman-plugins-3.2-nonscl-rhel7 foreman-plugins-3.2-rhel7 foreman-plugins-3.2-el8
 builder = tito.builder.FetchBuilder
 builder.rpmbuild_options = --define "foremandist .fm3_2"
 
 [koji-foreman-client]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.2-rhel7 foreman-client-3.2-el8 foreman-client-3.2-fedora29
 
 [koji-foreman-client-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.2-rhel7 foreman-client-3.2-el8 foreman-client-3.2-fedora29
```

Comment:
we don't release client for fedora29, and @pcreech wanted to clean up the config in develop for a while now